src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py

```diff
@@ -240,54 +247,83 @@ def load_arguments(self, _):
         c.argument('throughput', type=int, help='The throughput of SQL database (RU/s).')
         c.argument('max_throughput', max_throughput_type)
 
+    with self.argument_context('cosmosdb sql database throughput migrate') as c:
+        c.argument('account_name', account_name_type, id_part=None)
+        c.argument('database_name', options_list=['--name', '-n'], help="Database name")
+        c.argument('throughput_type', throughput_type)
+
     with self.argument_context('cosmosdb sql container throughput') as c:
         c.argument('account_name', account_name_type, id_part=None)
         c.argument('database_name', database_name_type)
         c.argument('container_name', options_list=['--name', '-n'], help="Container name")
         c.argument('throughput', type=int, help='The throughput of SQL container (RU/s).')
         c.argument('max_throughput', max_throughput_type)
 
+    with self.argument_context('cosmosdb sql container throughput migrate') as c:
+        c.argument('throughput_type', throughput_type)
```

Comment:
Could we consider using a loop to add this parameter to these commands?
For example:
```
    for scope in ['sql database throughput migrate', 'sql container throughput migrate' ... ]:
        with self.argument_context('cosmosdb {}'.format(scope)) as c:
            c.argument('throughput_type', throughput_type)
```

Reply:
I don't think this would add any major cleanliness in the code. I think the code is more readable in the current form, all the logically related  commands are placed close to each other, which is helpful when someone new reads the code. 

In short, I don't feel addition would contributes a lot to the code readability, I would prefer to keep it this way.

Reply:
I personally think that if these commands don't extend other arguments in the future, then using loop can help us quickly figure out which commands support argument `throughput_type` and make the code concise.
If these commands will extend other parameters later, I agree with you.